From 28e4cf5dab98e34c743330df19cee4a1edabb38a Mon Sep 17 00:00:00 2001
From: Jun Miyauchi <jun.miyauchi.sr@hitachi.com>
Date: Thu, 15 Feb 2018 15:33:20 +0900
Subject: [PATCH 602/909] arm64: dts: r8a77990: Add optee node

commit 302879e890a65b4dc30c877e73b646de7aacf759 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds op-tee node for the R8A77990 SoC.

Signed-off-by: Jun Miyauchi <jun.miyauchi.sr@hitachi.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990.dtsi b/arch/arm64/boot/dts/renesas/r8a77990.dtsi
index ebbc8e1d8ce8..8ee27c456550 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77990.dtsi
@@ -135,6 +135,13 @@
 		clock-frequency = <0>;
 	};
 
+	firmware {
+		optee {
+			compatible = "linaro,optee-tz";
+			method = "smc";
+		};
+	};
+
 	gsx_opp_table: gsx_opp_table {
 		compatible = "operating-points-v2";
 
-- 
2.17.1

